app/views/volunteers/edit.html.erb

```diff
@@ -54,6 +54,12 @@
           <div class="alert alert-danger">
             Volunteer was deactivated on: <%= @volunteer.updated_at.strftime("%m-%d-%Y") %>
           </div>
+          <% if current_user.role == "casa_admin" || @volunteer.supervisor == current_user %>
```

Comment:
This type of auth policy should go in a https://github.com/varvet/pundit policy file like `casa/app/policies/user_policy.rb`

Reply:
This is tracked in https://github.com/rubyforgood/casa/issues/441